tests/test_dispatchers.py

```diff
@@ -96,21 +96,25 @@ def test_productmap_with_all_arguments_mapped(func, args, grids, expected, reque
     expected = request.getfixturevalue(expected)
 
     decorated = productmap(func, args)
-    calculated_args = decorated(*grids.values())
-    calculated_kwargs = decorated(**grids)
+    calculated = decorated(**grids)
+    aaae(calculated, expected)
 
-    aaae(calculated_args, expected)
-    aaae(calculated_kwargs, expected)
+    match = (
```

Comment:
Shouldn't we add another test for this? I find it a bit confusing to have two in one. If it runs for very long and that is not an option, make a note?